pkg/apis/extensions/v1beta1/types.go

```diff
@@ -175,7 +175,7 @@ type ThirdPartyResourceData struct {
 	Data []byte `json:"data,omitempty"`
 }
 
-// Deployment enables declarative updates for Pods and ReplicationControllers.
+// Deployment enables declarative updates for Pods and ReplicationSets.
```

Comment:
That accidentally sneaked in. As you can you see, that was a half-hearted change (change to ReplicationSets instead of ReplicaSets) :P 

Sorry about that. Fixed.